samples/subsys/ipc/ipm_mhu_dual_core/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _ipm_mhu_dual_core:
+
+MHU Dual Core
+###########
+
+Overview
+********
+A simple dual core example that can be used with arm Musca A1
```

Comment:
After reading this description I have no idea what this sample does or what it's for (but you do mention some facts in the commit message that should be added here.

How about:
```
An MHU (Message Handling Unit) enables software to raise interrupts to
the processor cores.  This sample is a simple dual-core example for a
Musca A1 board that has two MHU units.
```

And then can you add some more explanation of what the sample does?

Reply:
Thanks, I have updated the documents in latest commit.